### ISSUE TITLE
Question - How can I execute The example compiled code

### ISSUE THREAD

**Issue by mohsenomidi** [CppMicroServices/CppMicroServices#355]
Dears, 

I Just install the CppMicroServices to my Linux box and i compiled successfully Source Code and also Examples now i want to start test to run GettingStart execution file. 

This is the path of execution file (after make it created.).
user@test:~/CppMicroServices-development/doc/src/bin$

The Libraries located in this path: (~/CppMicroServices-development/doc/src/lib)

Now i just execute the "GettingStarted" execution file (with and without parameter) :

`user@test:~/CppMicroServices-development/doc/src/bin$ ./GettingStarted`
`Pass shared libraries as command line arguments
Bundle#2 start failed: Bundle activator constructor not found`

`user@test:~/CppMicroServices-development/doc/src/bin$ ./GettingStarted ../lib/libServiceTime_SystemClock.a`
`Failed to install bundle library at ../lib/libServiceTime_SystemClock.a: Bundle#3, a bundle with same symbolic name and version is already installed (service_time_systemclock, 0.0.0)
Bundle#2 start failed: Bundle activator constructor not found`

i didnt change anything in make file, CMake File or etc... i just want execute the system sample to understand how it works, after that i will implement my own services, based on tutorial i follow the steps and read getting start part and i just follow the manual, installation, compilation done successfully (without any error), but execution failed. 

can you help me to start the samples

**Comment by jeffdiclemente** [comment]
@mohsenomidi It looks like you are building CppMicroServices and the Getting Started project with ```US_BUILD_SHARED_LIBS``` turned off, meaning no shared libraries are built and instead all bundles are built as static libraries. In this configuration, there is a bug in the Getting Started project. The getting started executable attempts to install bundles assuming they are shared libraries passed on the command line, even though all bundles were linked into the GettingStarted executable as static libraries (and passing them through the command line is irrelevant).

To get you unblocked, you can either:
1. modify ```doc/src/examples/getting_started/main.cpp``` as follows:
```
 #ifdef US_BUILD_SHARED_LIBS
  // Install all bundles contained in the shared libraries
  // given as command line arguments.
  for (int i = 1; i < argc; ++i) {
    try {
      ctx.InstallBundles(argv[i]);
    } catch (const std::exception& e) {
      std::cerr << e.what() << std::endl;
    }
  }
#endif
```
adding an #ifdef around the problematic piece of code.

2. Build CppMicroServices and all other projects with US_BUILD_SHARED_LIBS turned on and then pass in the shared library paths to the ```GettingStarted``` executable.
e.g. ```./GettingStarted ../lib/libServiceTime_SystemClock.so```

In the meantime, I will fix the bug.

**Comment by mohsenomidi** [comment]
@jeffdiclemente Thank you So Much... fingers crossed 
i works fine, i enabled BUILD_SHARED_LIBS and compiled the code now it seems okay, just I need study to understand how it works, if I have questions, I will come back to you :)

**Comment by jeffdiclemente** [comment]
@mohsenomidi good to hear its working for you now. I'm going to close this issue. If you have any other issues or questions, please feel free to open up another issue. Thanks.
